protobuf/build.gradle

```diff
@@ -15,6 +26,8 @@ dependencies {
     }
 }
 
+configureProtoCompilation()
```

Comment:
I think we've been putting services in the `services/` project (and the `io.grpc.protobuf.service` package).